rpcs3/Emu/RSX/Overlays/overlay_list_view.cpp

```diff
@@ -2,6 +2,8 @@
 #include "overlay_controls.h"
 #include "Emu/system_config.h"
 
+#include <QCoreApplication>
```

Comment:
Why are you suddenly adding random Qt translations to the emu core?
I don't even know if we want Qt anywhere outside of the frontend.
It won't work anyway if you don't reinitialize the code after a language change.
It would make more sense to get the stuff with an emu callback from some translation object (localized or tooltip for example)

Reply:
You could for example add a Emu.callback that passes an enum class value and returns a std string.
The enum could be defined in its own header in the emu utile.
Then you can import that into rsx or anywhere else and then just have the actual callback return values by map from a localized object.